pages/Configuring/Keywords.md

```diff
@@ -160,12 +160,18 @@ layerrule = blur,address:0x<ADDRESS>
 
 you can get the namespace / address from `hyprctl layers`.
 
-To remove a blurls rule (useful in dynamic situations) use:
+To remove a layer rule (useful in dynamic situations) use:
 
 ```ini
 layerrule = unset,<whatever you used before>
 ```
 
+For example:
+
+```ini
+layerrule = unset,blur,NAMESPACE
```

Comment:
`layerrule = unset,NAMESPACE`

Reply:
that's exactly the confusion point I was trying to fix :)